src/test/resources/au/org/ala/bie/metadata-1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Some test metadata",
+  "description": "A test description",
+  "version": "1.1",
```

Comment:
Do we need version? There is no history of metadata.

Reply:
These are set up so that we can have an archive of versions on archives.ala.org.au. Although not now, so that we can decide whether it looks like an insult or not.

The metadata, particularly the version/created/modified is optional but means that we can track stuff floating about on the web.

Reply:
If we can save the history, e.g. on archives.ala.org.au, then they are good to have.